### PR TITLE
#5145 [Manager] Add a column to indicate the availability of the stre…

### PR DESCRIPTION
**Pull request by eechih**
…am jar

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5145 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Workspace/Settings/pages/StreamJarsPage.js [hunk 1]
```diff
@@ -184,6 +199,25 @@ function StreamJarsPage() {
                 );
               },
             },
+            {
+              title: 'Valid',
+              render: file => {
+                const isValidStreamJar = some(
+                  file?.classInfos,
+                  classInfo => classInfo?.classType === 'stream',
```
Comment (saivirtue) [inline ohara-manager/client/src/components/Workspace/Settings/pages/StreamJarsPage.js:207]: The classType is as same as the `kind` in const.ts :)

Comment (eechih) [reply ohara-manager/client/src/components/Workspace/Settings/pages/StreamJarsPage.js:207]: Get it. I will update :)

**Comment by eechih** [comment]
retry manager-e2e

File: ohara-manager/client/src/components/Workspace/Settings/pages/StreamJarsPage.js [hunk 2]
```diff
@@ -28,13 +29,28 @@ import {
...
 import { getKey } from 'utils/object';
 import WorkspaceFileSelectorDialog from '../common/WorkspaceFileSelectorDialog';
 
+export const StyledCheckIcon = styled(CheckIcon)(
+  ({ theme }) => css`
+    color: ${theme.palette.success.main};
+  `,
+);
+
+export const StyledWarningIcon = styled(WarningIcon)(
```
Comment (konekoya) [inline ohara-manager/client/src/components/Workspace/Settings/pages/StreamJarsPage.js:48]: Do we need to export these? Are they using in other components 🤔

Comment (eechih) [reply ohara-manager/client/src/components/Workspace/Settings/pages/StreamJarsPage.js:48]: @konekoya 目前只有這個 Table 會用到，未來如果有其它 component 也要用的話，再考慮抽到 common。你覺得如何呢？

Comment (eechih) [reply ohara-manager/client/src/components/Workspace/Settings/pages/StreamJarsPage.js:48]: 經過討論後，決定做一個 HOC 來改變 icon color。

範例
```
<IconWrapper color="warning">
  <WarningIcon />
</IconWrapper>
```

Comment (konekoya) [reply ohara-manager/client/src/components/Workspace/Settings/pages/StreamJarsPage.js:48]: Sounds good for me 👍

Comment (eechih) [reply ohara-manager/client/src/components/Workspace/Settings/pages/StreamJarsPage.js]: @konekoya I created an `IconWrapper` HOC to change the color of the icon. Please review again, thanks.

**Comment by eechih** [comment]
retry manager-e2e

**Comment by eechih** [comment]
retry manager-e2e

File: ohara-manager/client/src/components/Workspace/Settings/pages/StreamJarsPage.js (hunk 2 above)
Comment (konekoya) [reply ohara-manager/client/src/components/Workspace/Settings/pages/StreamJarsPage.js]: Looks good to me. Thanks for putting this together 👍

**Pull request merged**